docs/integrations.md

```diff
@@ -88,6 +85,11 @@ jobs:
       #  if: always()
 ```
 
+#### Inline PR Comments (beta)
+Set `GITHUB_TOKEN` (commented out in the code above) to leave inline comments on your pull requests.
```

Comment:
@sabihb Could we add a `!!! info` notice that this feature is currently for GitHub users only?

When you say "commented out in the code above", what code above has it commented out? Do all of the providers have that code?

Reply:
👍  Will add the notice! I'll clarify the sentence about the commented out code. I was attempting to refer to the yaml file example above this paragraph (where the two lines to add inline pr comments are commented out).

Reply:
Updated! Let me know what you think.